common/setups/rasr/util/rasr.py

```diff
@@ -198,5 +198,17 @@ def get_step_iter(self):
     def get_step_names_as_list(self):
         return list(self._step_names_args.keys())
 
+    def get_non_gmm_steps_as_list(self):
+        return ["forced_align"]
+
+    def get_gmm_steps_names_as_list(self):
+        step_names = self.get_step_names_as_list()
+        # TODO nicer list comprehension
+        for non_gmm_step in self.get_non_gmm_steps_as_list():
+            step_names = [
+                name for name in step_names if not name.startswith(non_gmm_step)
+            ]
+        return list(self._step_names_args.keys())
```

Comment:
this is the same as in the function `get_step_names_as_list`. did you mean
```suggestion
        return step_names
```